15_totalIntegers/solution/totalIntegers-solution.js

```diff
@@ -0,0 +1,15 @@
+const totalIntegers = function(obj, count = 0) {
+
+  if (typeof obj !== 'object' || obj === null) return;
+
+  const elements = Array.isArray(obj) ? obj : Object.values(obj);
```

Comment:
```suggestion
  const elements = Object.values(obj);
```
Object.values() coerces to an object, so passing it an array will just have `Object.values(obj)` return the array itself.

Reply:
Ohh right, that's neat!